binary_transparency/firmware/common/key_info.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+//RSApri is a variable to store the pem formatted private key
+var RSApri string = `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
bump the PEM header down onto the next line to match the one below?

Also, these can be `const` I think

Reply:
ACK - for both